repository/kafka/README.md

```diff
@@ -7,3 +7,12 @@ The KUDO Kafka operator creates, configures and manages [Apache Kafka](https://k
 The latest stable version of Kafka operator is `0.1.1`
 For more details, please see the [docs](./docs/v0.1) folder.
 
+For the latest master branch you can check  [docs](./docs/latest) docs 
+
+
+## Version Chart
+
+| KUDO Kafka Version | Apache Kafka Version |
+| ------------------ | -------------------- |
+| 0.1.1              | 2.2.1                |
+| master             | 2.3.0                |
```

Comment:
should we rather say latest here? or rather explicitly say 0.2.0 ?

Reply:
`latest` over `0.2.0` as we can install the operator as it is in master. Which may not be the last stable release.